src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.tsx

```diff
@@ -83,25 +83,20 @@ export const TranscriptsListItemInfo = (
     objectId: props.gene.unversioned_stable_id
   });
 
-  const getLinkToProteinView = (transcriptId: string) => {
-    if (transcript.so_term !== 'protein_coding') {
-      // TODO: Use Protein ID
-      return transcriptId;
-    }
+  const getLinkToProteinView = (stableId: string) => {
     const proteinViewUrl = urlFor.entityViewer({
       genomeId,
       entityId,
       view: View.PROTEIN,
-      transcriptId
+      transcriptId: stableId
```

Comment:
So, here, for example, I am already confused. Why is the `transcriptId` field receiving the protein id?

Reply:
Here and below, I have changed it all to proteinId. I was planning to update it today as it was getting late yesterday.